src/components/career-form/career-form.tsx

```diff
@@ -78,6 +106,7 @@ export const CareerForm: React.FC<CareerFormProps> = (props) => {
       .then((json) => {
         if (json.success) {
           // all good
+          props.scrollToStart();
```

Comment:
nice. sometimes I think we should not do this, because the user is the owner of the scrollbar. But I have seen Fabian's comment and I think it's okay here to do that.